pkg/reconciler/openshift/tektonresult/extension.go

```diff
@@ -178,6 +180,34 @@ func filterAndTransform() client.FilterAndTransform {
 	}
 }
 
+func injectResultsAPIServiceCACert(props v1alpha1.ResultsAPIProperties) mf.Transformer {
+	return func(u *unstructured.Unstructured) error {
+		if u.GetKind() != "Service" || u.GetName() != serviceAPI {
+			return nil
+		}
+
+		s := &corev1.Service{}
+		err := k8sruntime.DefaultUnstructuredConverter.FromUnstructured(u.Object, s)
+		if err != nil {
+			return err
+		}
+
+		annotations := s.Annotations
+		if annotations == nil {
+			annotations = make(map[string]string)
+		}
+		annotations["service.beta.openshift.io/serving-cert-secret-name"] = "tekton-results-tls"
```

Comment:
what is `"tekton-results-tls"` here

Reply:
That's the TLS cert name. Yes, it is always same.  Maybe we can put this in a constant. No need to configure this.

Reply:
So added this to a constant.